README.md

```diff
@@ -220,6 +221,45 @@ const invalidErr = new Error('foobar');
 isValidationErrorLike(err); // returns false
 ```
 
+### isZodErrorLike
+
+A [type guard](https://www.typescriptlang.org/docs/handbook/2/narrowing.html#using-type-predicates) utility function, based on _heuristics_ comparison.
+
+_Why do we need heuristics since we can use a simple `instanceof` comparison?_ Because of multi-version inconsistencies. For instance, it's possible that a dependency is using an older `zod` version internally. In such case, the `instanceof` comparison will yield invalid results because module deduplication does not apply at npm/yarn level and the prototype is different.
+
+#### Arguments
+
+- `error` - error instance (required)
+
+#### Example
+
+```typescript
+import { z as zod } from 'zod';
+import { ValidationError, isZodErrorLike } from 'zod-validation-error';
+
+const zodValidationErr = new ValidationError('foobar');
+isZodErrorLike(zodValidationErr); // returns false
+
+const genericErr = new Error('foobar');
+isZodErrorLike(genericErr); // returns false
+
+// create zod schema
+const zodSchema = zod.object({
+  id: zod.number().int().positive(),
+  email: zod.string().email(),
+});
+
+// parse some invalid value
+try {
+  zodSchema.parse({
+    id: 1,
+    email: 'foobar', // note: invalid email value
+  });
+} catch (err) {
+  isZodErrorLike(err); // returns true
+}
```

Comment:
@thanoskrg would it make more sense to replace all of this complexity with a simple...

```
const err = new zod.ZodError([
  {
    code: zod.ZodIssueCode.custom,
    path: [],
    message: 'foobar',
    fatal: true,
  },
]);
```

It's just that users don't typically instantiate `ZodErrors` themselves, so I am thinking some people might be confused.

Reply:
Sure, why not?